pkgs/applications/radio/gridtracker/default.nix

```diff
@@ -1,14 +1,20 @@
-{ lib, stdenv, fetchFromGitLab, nix-update-script, nwjs, wrapGAppsHook }:
+{ lib,
+  stdenv,
+  fetchFromGitLab,
+  nix-update-script,
+  nwjs,
+  wrapGAppsHook
+}:
 
 stdenv.mkDerivation rec {
   pname = "gridtracker";
-  version = "1.23.0402";
+  version = "1.23.1019";
```

Comment:
there is an existing pull request for this version at https://github.com/NixOS/nixpkgs/pull/261122